test/extensions/quic_listeners/quiche/platform/http2_platform_test.cc

```diff
@@ -96,6 +99,20 @@ TEST(Http2PlatformTest, Http2StringPiece) {
   EXPECT_EQ('b', sp[0]);
 }
 
+TEST(Http2PlatformTest, Http2Flags) {
+  auto& flag_registry = quiche::FlagRegistry::GetInstance();
+  flag_registry.ResetFlags();
+  EXPECT_FALSE(GetHttp2ReloadableFlag(http2_testonly_default_false));
+  SetHttp2ReloadableFlag(http2_testonly_default_false, true);
+  EXPECT_TRUE(GetHttp2ReloadableFlag(http2_testonly_default_false));
+
+  flag_registry.ResetFlags();
+  EXPECT_FALSE(GetHttp2ReloadableFlag(http2_testonly_default_false));
+  flag_registry.FindFlag("http2_reloadable_flag_http2_testonly_default_false")
```

Comment:
nit: Would be nice if we can cover all the possible values("true", "t", "no", "n", ...) to set a boolean flag.

Reply:
Sure, done.